### PR TITLE
Public

### PR DESCRIPTION
**Pull request by HourglassMagi**
Made some changes to my App component to hold state.

### CODE REVIEW

**Comment by HourglassMagi** [comment]
[PR Laurent update to public folder and react app component](https://trello.com/c/HJAR4hZr/46-pr-laurent-update-to-public-folder-and-react-app-component)

File: package.json [hunk 1]
```diff
@@ -8,10 +8,20 @@
...
+    "pg": "^7.4.1",
+    "react": "^16.3.2",
+    "react-dom": "^16.3.2"
   },
   "devDependencies": {
-    "eslint-config-hackreactor": "git://github.com/reactorcore/eslint-config-hackreactor"
+    "babel-cli": "^6.26.0",
+    "babel-loader": "^7.1.4",
+    "eslint-config-hackreactor": "git://github.com/reactorcore/eslint-config-hackreactor",
+    "webpack": "^4.6.0",
```
Comment (careyl96) [inline package.json]: webpack should be in regular dependencies

File: public/dist/style.css [hunk 2]
```diff
@@ -0,0 +1,5 @@
+body {
+  font-family: Arial, Helvetica, sans-serif;
+  background-color: rgb(51, 108, 126) ;
+ color: white;
```
Comment (careyl96) [inline public/dist/style.css]: indentation here

File: public/dist/style.css [hunk 3]
```diff
@@ -0,0 +1,5 @@
+body {
+  font-family: Arial, Helvetica, sans-serif;
+  background-color: rgb(51, 108, 126) ;
```
Comment (careyl96) [inline public/dist/style.css]: remove space before semicolon

File: public/src/components/App.jsx [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+import Axios from 'axios';
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      recipe: []
+    };
+    this.getRecipe = this.getRecipe.bind(this)
+  };
```
Comment (careyl96) [inline public/src/components/App.jsx]: space after this line

File: public/src/components/App.jsx [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+  getRecipe() {
+    Axios.get('http://127.0.0.1:3100/recipe')
+    .then((res) => {
+      console.log('this is the response', res.data);
+    })
+
+  }
```
Comment (careyl96) [inline public/src/components/App.jsx]: remove space before this line and add one after

File: public/src/components/App.jsx [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+    this.state = {
+      recipe: []
+    };
+    this.getRecipe = this.getRecipe.bind(this)
+  };
+  getRecipe() {
+    Axios.get('http://127.0.0.1:3100/recipe')
+    .then((res) => {
+      console.log('this is the response', res.data);
+    })
```
Comment (careyl96) [inline public/src/components/App.jsx]: missing semicolon

File: public/src/components/App.jsx [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      recipe: []
+    };
+    this.getRecipe = this.getRecipe.bind(this)
+  };
+  getRecipe() {
+    Axios.get('http://127.0.0.1:3100/recipe')
```
Comment (careyl96) [inline public/src/components/App.jsx]: your route should be more specific than just /recipe; should be /recipe/:id/instructions or something similar

File: public/src/components/App.jsx [hunk 8]
```diff
@@ -0,0 +1,29 @@
...
+
+  }
+  render() {
+    return (
+      <div>
+        <h1>Instructions</h1>
+        <h2>good boy</h2>
+        <button onClick={this.getRecipe}>click me</button>
+      </div>
+    )
```
Comment (careyl96) [inline public/src/components/App.jsx]: missing semicolon

File: public/src/index.jsx [hunk 9]
```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+import ReactDom from 'react-dom';
+import App from './component/App';
```
Comment (careyl96) [inline public/src/index.jsx]: component should be components, unless you're only planning on putting one component in this folder

**Review by careyl96 — CHANGES_REQUESTED** [review]

**Comment by HourglassMagi** [comment]
@CareyL96  I made the requested changees, webpack is in dev dependencies because I am using a webpack dev-server for my client. let me know if you see anything else. I also updated my package.json to reflect that.

**Review by careyl96 — APPROVED** [review]
looks good!

**Pull request closed without merging**